src/instance.ts

```diff
@@ -192,6 +238,249 @@ class Instance extends common.GrpcServiceObject {
+    });
+    this.request<IBackup[]>(
+      {
+        client: 'DatabaseAdminClient',
+        method: 'listBackups',
+        reqOpts,
+        gaxOpts: query,
+      },
+      (err, rowBackups, ...args) => {
+        let backups: Backup[] | null = null;
+        if (rowBackups) {
+          backups = rowBackups.map(rowBackup => {
+            const rowBackupName = rowBackup.name!;
+            const backupId = rowBackupName.substring(
+              rowBackupName.lastIndexOf('/') + 1
+            );
+            return this.backup(
+              backupId,
+              rowBackup.database!,
+              new PreciseDate(rowBackup.expireTime as DateStruct)
+            );
+          });
+        }
+
+        callback(err, backups, ...args);
+      }
+    );
+  }
+
+  getBackupOperations(
+    query?: GetBackupOperationsRequest
+  ): Promise<GetBackupOperationsResponse>;
+  getBackupOperations(callback: GetBackupOperationsCallback): void;
+  getBackupOperations(
+    query: GetBackupOperationsRequest,
+    callback: GetBackupOperationsCallback
+  ): void;
+  /**
+   * Query object for listing backup operations.
+   *
+   * @typedef {object} GetBackupOperationsRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetBackupOperationsResponse
+   * @property {IOperation[]} 0 Array of {@link IOperation} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * List pending and completed backup operations for all databases in the instance.
+   *
+   * @see {@link #listOperations}
+   *
+   * @param query query object for listing backup operations.
+   * @returns {Promise<GetBackupOperationsResponse>} when resolved, contains a
+   *     paged list of backup operations.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const [operations] = await instance.getBackupOperations();
+   */
+  getBackupOperations(
+    queryOrCallback?: GetBackupOperationsRequest | GetBackupOperationsCallback,
+    cb?: GetBackupOperationsCallback
+  ): void | Promise<GetBackupOperationsResponse> {
+    const callback =
+      typeof queryOrCallback === 'function' ? queryOrCallback : cb!;
+    const query =
+      typeof queryOrCallback === 'object'
+        ? queryOrCallback
+        : ({} as GetBackupOperationsRequest);
+
+    const reqOpts = extend({}, query, {
+      parent: this.formattedName_,
+    });
+    this.request<IOperation[]>(
+      {
+        client: 'DatabaseAdminClient',
```

Comment:
Same idea as my previous comment about adding this to the Database class as well, where only backup operations from the database are returned.

Reply:
Same reasoning as above. While backups get created for a database, the backup and the operation belongs to the instance. So we were asked to put this in the instance since you can get all backup operations in the instance.

Reply:
Thank you!